script.audio.profiles/resources/lib/profiles.py

```diff
@@ -0,0 +1,252 @@
+# -*- coding: utf-8 -*-
+# *  Credits:
+# *
+# *  original Audio Profiles code by Regss
+# *  updates and additions through v1.4.1 by notoco and CtrlGy
+# *  updates and additions since v1.4.2 by pkscout
+
+from kodi_six import xbmc, xbmcaddon, xbmcvfs
+import json, os, sys
+import resources.lib.dialog as dialog
+import resources.lib.notify as notify
+
+ADDON = xbmcaddon.Addon()
```

Comment:
all these variables are used everywhere, maybe a good idea to move it to a common file where you can just import ? or define them in __all__